src/cmd/cmd_replay.ml

```diff
@@ -298,8 +298,13 @@ let cmd ~unsafe ~optimize ~replay_file ~source_file ~entry_point
                    (Fmt.str "unexpected value type: %a" Smtml.Value.pp v) ) )
           bindings
       in
-      Array.of_list model
+      (entry_point, Array.of_list model)
   in
+  let entry_point =
+    if Option.is_none entry_point then model_entry_point else entry_point
+  in
+  Logs.debug (fun m -> m "%s" (Option.value ~default:"na" entry_point));
+  Logs.debug (fun m -> m "%s" (Option.value ~default:"na" model_entry_point));
```

Comment:
Maybe add more information about what you're printing? Something like `m "entry point: %s" ...`